### DISCUSSION TITLE
The navigationsub snippet breaks the Knowhow template

### DISCUSSION THREAD

**Discussion by dajare** [datenstrom/community#63] [Report a bug] [answered]
Hi! I just tried using the `navigationsub` snippet with the Knowhow template. It loads the main nav, then breaks. I tried commenting out the `navigationsearch` snippet in case there was a conflict, but nothing changed: the nav loaded, then broke.

I have the `navigationsub` snippet working with a different theme - so is there some conflict with Knowhow? Would be good to get that working. Thanks!

**Comment by markseuffert** [comment]
Hi David! Yes, you are right. I think the problem is that most snippets don't work without adjustments. I tested the `navigationsub` snippet and it doesn't create a working navigation "out of the box".  It needs additional CSS, probably a different one for each theme. 

Perhaps snippets should be better integrated with the rest? Or removed if they don't come as a complete working package? What do you think?

**Comment by markseuffert** [comment]
I think the navigation snippets should just work. Not good if it's difficult for people.

**Comment by markseuffert** [comment, marked as answer]
I think I have it working. The navigation snippets are now part of the Yellow installation. I renamed `navigationsub` to `navigation-sidebar` and tested that it works with any theme. 

Please let me know if this works for you. Thanks.

**Comment by dajare** [comment]
Here's what I did:
- created the `navigation-sidebar` snippet 
- added it to the `knowhow.php` template
- and ... borked. :( 

Do I need to do a fresh install? My "knowhow" theme CSS says version  0.1.2, and I see current is 5.5! Will update this comment after I do some more tweaking.

**Comment by nibreh** [comment]
hi ! 

You must update yellow , themes and plugins, and use `navigation-sidebar` at the top of your page or in `config.ini` for all pages. 

https://github.com/datenstrom/yellow/wiki/Yellow-customisation#custom-navigation

**Comment by dajare** [comment]
@nibreh - thanks! So many changes (all for the good!) since I last "tweaked". Got a result now. :tada:
